website/docs/layers/prometheus.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Prometeus
+---
+
+# Introduction to Prometheus
```

Comment:
Those docs should be added in `PrometheusLayer` struct comments. OpenDAL follows the code as document style.